server.key

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I see - my mistake. Reverted the delete; these files exist again.

Did you have any luck running it as ssl?